app/src/main/kotlin/io/element/android/x/initializer/TracingInitializer.kt

```diff
@@ -19,7 +19,7 @@ import io.element.android.libraries.core.meta.BuildType
 import io.element.android.libraries.matrix.api.tracing.TracingConfiguration
 import io.element.android.libraries.matrix.api.tracing.TracingFilterConfigurations
 import io.element.android.libraries.matrix.api.tracing.WriteToFilesConfiguration
-import io.element.android.x.BuildConfig
+import org.therevivalists.app.BuildConfig
```

Comment:
can't we keep the original package name in dirs and just change the apk's package name?

Reply:
BuildConfig is generated on the fly based on the package name